lib/shopify_transporter/exporters/magento/customer_exporter.rb

```diff
@@ -38,6 +43,20 @@ def base_customers
         def customer_address_list(customer_id)
           @client.call(:customer_address_list, customer_id: customer_id).body
         end
+
+        def print_customer_details_error(customer, e)
```

Comment:
Do you anticipate the error handling to change for these?

The output looks very similar for all these objects, so I'm wondering if we should extract the method to a helper class. I don't feel too strongly about this, so as is is fine as well.

Reply:
How's moving it to the base exporter class?

Reply:
I'm okay with that - would we lose functionality that way?

Reply:
Yeah, it'd end up being less friendly. I went ahead and tried that out but realized it actually worsens the experience. I prefer this slightly less DRY approach that can easily be changed later /shrug.

Reply:
agree w fahia